src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorResolver.cs

```diff
@@ -121,6 +125,7 @@ public IEnumerable<TagHelperDescriptor> Resolve([NotNull] TagHelperDescriptorRes
         protected virtual IEnumerable<TagHelperDescriptor> ResolveDescriptorsInAssembly(
             string assemblyName,
             SourceLocation documentLocation,
+            bool designTime,
```

Comment:
Remove this parameter. Any subclass must provide value when calling a `TagHelperDescriptorResolver` constructor. So that subclass can have its own `private bool _designTime` field.